rust/test

```diff
@@ -1478,6 +1490,35 @@ test_internal_commands() {
     teardown
 }
 
+# Indexer correctly starts from config
+test_start_from_config() {
+    test=test_start_from_config
+
+    setup
+    idxr_server_start \
+        --blocks-dir ./blocks \
+        --ledgers-dir ./staking-ledgers \
+        --database-dir ./database \
+        --log-dir ./logs \
+        --config-file ./logs/config.json
+    wait_for_socket
+    teardown
+
+    setup
+    dl_mainnet 15 ./blocks
+
+    idxr_server config -p ./config.json
```

Comment:
So, this means that specifying a config also means "server start" subcommands?

Reply:
I'm not 100% set on the name, but those are the semantics I was going for

Reply:
Okay. If those are the semantics, then the name of the switch as-is will confuse users. I recommend changing it to something like "start-with-config-file".